_rules/SC4-1-2-aria-attribute-has-valid-value.md

```diff
@@ -0,0 +1,76 @@
+---
+name: ARIA attribute has valid value
+description: |
+  This rule checks that `aria-*` attributes have a valid value
+  
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any non-empty attribute that is placed on an HTML or SVG element, and where the attribute name starts with `aria-`.
+
+### Expectation
+
+Each of the applicable attributes have a valid value according to the [value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value) for the given state or property (`aria-*` attribute) as specified by [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+**Note:** 
+- If the value of the `aria-*` attribute is of value type `ID Reference` or `ID Reference List` ensure that an element with the given id exists in the document. 
```

Comment:
I don't think this should be required. I've seen things like `aria-errormessage` being used statically, and relying on the presence or absence of an element as a way to toggle the errormessage. There are legitimate use cases for defining non-existing elements for many ARIA idref attributes. The only thing there is that an `ID Reference` isn't allowed to contain spaces.

Reply:
Note now says this is out of scope.